### PR TITLE
Created bookings seed

### PR DESCRIPTION
**Pull request by eztree**
## Why

This pull request is needed because:

Had to create bookings seed

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,10 +5,11 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-puts 'Destroying users and centers..'
+puts 'Destroying everything... 💣'
 Listing.destroy_all
 Center.destroy_all
 User.destroy_all
+Booking.destroy_all
```
Comment (JensenYuen) [inline db/seeds.rb]: Booking destroy before listing destroy

File: db/seeds.rb [hunk 2]
```diff
@@ -54,4 +55,16 @@
   )
 end
 
+5.times do
+  puts "Creating a booking now 📚"
+  booking = Booking.create!(
+    user_id: rand(1..10),
```
Comment (JensenYuen) [inline db/seeds.rb]: user: User.first

File: db/seeds.rb [hunk 3]
```diff
@@ -54,4 +55,16 @@
...
+    listing_id: rand(1..10),
```
Comment (JensenYuen) [inline db/seeds.rb]: listing: Listing.first

**Review by JensenYuen — CHANGES_REQUESTED** [review]
Added comments on code

File: db/seeds.rb (hunk 1 above)
Comment (eztree) [reply db/seeds.rb]: THANK U

**Review by JensenYuen — APPROVED** [review]
all good

**Pull request merged**